browser/brave_content_browser_client.cc

```diff
@@ -555,6 +557,12 @@ void BraveContentBrowserClient::RegisterWebUIInterfaceBrokers(
   if (base::FeatureList::IsEnabled(skus::features::kSkusFeature)) {
     registry.ForWebUI<SkusInternalsUI>().Add<skus::mojom::SkusInternals>();
   }
+
+  if (base::FeatureList::IsEnabled(
```

Comment:
Sorry, this is still in the prototyping phase, so nothing is as clean as it probably should be. In addition, it might be useful to make this page available on Android, as it can be handy for debugging the feed (but we haven't decided yet).